portfolio/src/main/webapp/script.js

```diff
@@ -65,13 +65,16 @@ async function fetchContentFromServlet() {
     const response = await fetch('/data');
     const arr = await response.json();
 
-    const stringList = document.getElementById('week2-fetch');
-    stringList.innerHTML = '';
+    const commentsList = document.getElementById('week2-fetch');
```

Comment:
You can choose a more description name than 'week2-fetch', a name that describes the meaning, for example, call it 'comment-div' tell more information than 'week2-fetch'.